docs/source/components/nodes/stereo_depth.rst

```diff
@@ -162,7 +162,8 @@ Currently configurable blocks
 Current limitations
 ###################
 
-When using subpixel mode, if both :code:`depth` and :code:`disparity` are used, only :code:`depth` will have valid output.
+- When using subpixel mode, if both :code:`depth` and :code:`disparity` are used, only :code:`depth` will have valid output.
```

Comment:
This is not applicable anymore, both depth and disparity are valid.

Reply:
```suggestion
```